src/components/Header/Header.js

```diff
@@ -1,18 +1,66 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import styled from 'styled-components';
+import { DiffAdded } from 'styled-icons/octicons/DiffAdded';
 
 import { ACTIVITY_PLAN_ROUTE, CREATE_ACTIVITY_ROUTE } from '../../constants/routes';
+import ActivitiesListFilter from '../ActivitiesListFilter';
+import { spacing, colors } from '../../styles/base/base';
+
+const HeaderWrapper = styled.header`
+  align-items: center;
+  display: flex;
+  height: 10vh;
+  justify-content: space-around;
+
+  @media (min-width: ${spacing.desktopBreakpoint}) {
+    justify-content: space-between;
+  }
+`;
+
+const AppName = styled(NavLink)`
+  color: ${colors.black};
+  font-family: Satisfy;
+  font-size: 8vw;
```

Comment:
no, no-> font-size depending on the size of . the screen is totally wrong... Look -> at this moment you are setting the font size to 0.08 width of the screen. 
What if user will rotate the phone to horizontal position ? :O